src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,60 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { FilterStatus } from '../../types/filter';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  changeStatusOfTodos: (value: FilterStatus) => void;
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleClearSearchBtn: () => void;
+  query: string;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({
+  changeStatusOfTodos,
+  handleQueryChange,
+  handleClearSearchBtn,
+  query,
+}) => {
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    changeStatusOfTodos(event.target.value as FilterStatus);
+  };
+
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select data-cy="statusSelect" onChange={handleChange}>
+            <option value={FilterStatus.All}>All</option>
```

Comment:
It looks better with `All`, but tests require `all`:

![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/86923089/f03c2b0f-4a58-422f-9dd5-c7e5c376c776)

Change it to pas tests